src/st/cen/D600.c

```diff
@@ -177,17 +231,30 @@ INCLUDE_ASM("config/../asm/st/cen/nonmatchings/D600", func_801948EC);
 
 INCLUDE_ASM("config/../asm/st/cen/nonmatchings/D600", func_80194AD4);
 
-INCLUDE_ASM("config/../asm/st/cen/nonmatchings/D600", func_80194B34);
+s32 func_80194B34(u8 arg0, s16 arg1) { return D_80180BBC[arg0] * arg1; }
 
-INCLUDE_ASM("config/../asm/st/cen/nonmatchings/D600", func_80194B60);
+s16 func_80194B60(u8 arg0) { return D_80180BBC[arg0]; }
 
 INCLUDE_ASM("config/../asm/st/cen/nonmatchings/D600", func_80194B7C);
 
 u8 func_80194BE8(s16 x, s16 y) { return ((ratan2(y, x) >> 4) + 0x40); }
 
-INCLUDE_ASM("config/../asm/st/cen/nonmatchings/D600", func_80194C20);
+u8 func_80194C20(ObjInit2* arg0, ObjInit2* arg1) {
```

Comment:
I do not think the parameters of this one are `ObjInit2` but rather `Entity`. The reason is that X and Y are most likely to be coordinates and not `zPriority` and `palette`.